pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1030,6 +1030,22 @@ let
         };
       };
 
+      danielgavin.ols = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          publisher = "danielgavin";
+          name = "ols";
+          version = "0.1.26";
```

```suggestion
          version = "0.1.28";
```
https://marketplace.visualstudio.com/items?itemName=DanielGavin.ols